libr/flag/flag.c

```diff
@@ -719,6 +723,7 @@ R_API RFlagItem *r_flag_set_next(RFlag *f, const char *name, ut64 off, ut32 size
 			RFlagItem *fi = r_flag_set (f, newName, off, size);
 			if (fi) {
 				free (newName);
+				f->is_dirty = true;
```

Comment:
this function calls r_flag_set which already sets the dirty bit. so rm this line
```suggestion
```